cmd/kpod/logout.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containers/image/pkg/docker/config"
+	"github.com/kubernetes-incubator/cri-o/libpod/common"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	logoutFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "infile, i",
+			Usage: "Path of the authentication file. Default is XDG_RUNTIME_DIR/containers/auth.json",
```

Comment:
I think you should add
${XDG_\RUNTIME_DIR}

Reply:
fixed